src/modules/refresh.js

```diff
@@ -0,0 +1,22 @@
+import fetchValue from './fetchApi.js';
+
+const refreshData = async () => {
+  const refresh = document.querySelector('.refresh');
+  const value = await fetchValue();
+  const table = document.querySelector('.score-table');
+  table.innerHTML = '';
+  let placeholder = '';
+  value.forEach((score) => {
+    placeholder += `
+    <tr>
+    <td>${score.user}: ${score.score}</td>
+    </tr>
+    `;
+  });
+  table.innerHTML = placeholder;
+  refresh.addEventListener('click', () => {
+    refreshData();
```

Comment:
- [ ] since the refreshData function is an async function it would be nice if you await it when calling it and await can only work inside async, so your code should look like this 
```
 refresh.addEventListener('click', async () => {
    await refreshData();
...
```